### PR TITLE
Visiting fellows july2016

### PR DESCRIPTION
**Pull request by fjp2cornell**
For review on featuretest

### CODE REVIEW

**Comment by fjp2cornell** [comment]
This branch is now ready for merging

File: docroot/sites/all/themes/ilr_theme/js/ilr_theme.js [hunk 1]
```diff
@@ -372,4 +372,35 @@
       });
     }
   };
+
+  Drupal.behaviors.change_hover_color = {
+    attach: function (context, settings) {
+      $(window).load(function() {
+        var $selector = settings.change_hover_color.selector;
+        console.log($selector);
```
Comment (a-fro) [inline docroot/sites/all/themes/ilr_theme/js/ilr_theme.js]: Oops! Please remove the console.log.

File: docroot/sites/all/modules/features/ilr_experience_reports/ilr_experience_reports.module [hunk 2]
```diff
@@ -7,18 +7,12 @@
...
-  if ($theme == 'ilr_theme') {
-    drupal_add_js(drupal_get_path('module', 'ilr_experience_reports') . '/ilr_experience_reports.js', array(
-        'type' => 'file',
-        'scope' => 'footer'
-      ));
-
-    $inview_js = libraries_get_path('inview') . '/jquery.inview.min.js';
-    drupal_add_js($inview_js, array('type' => 'file', 'scope' => 'footer'));
+function ilr_experience_reports_node_view($node, $view_mode, $langcode) {
+  $selector = 'node-type-experience-report';
```
Comment (a-fro) [inline docroot/sites/all/modules/features/ilr_experience_reports/ilr_experience_reports.module]: Can you please move the `$selector` line in this and in ilr_visiting_fellows.module inside the `if` statement?

**Comment by a-fro** [comment]
@fjp2cornell The visiting_fellows.scss file is basically a cut and paste of experience_reports.scss. Were you planning to refactor this now?

**Pull request merged**